src/LogExpFunctions.jl

```diff
@@ -2,6 +2,7 @@ module LogExpFunctions
 
 using DocStringExtensions: SIGNATURES
 using Base: Math.@horner, @irrational
+using LinearAlgebra: LinearAlgebra
```

Comment:
This is weird. Is this what you want?
```suggestion
import LinearAlgebra
```

Reply:
It's how BlueStyle and JuliaFormatter require you to write it (`using X: X` instead of `import X`). I am just involved in too many projects that use BlueStyle currently and therefore automatically did it here as well, even though there's no style enforced. Do you want me to change it?

Reply:
Sure, I don't mind.